src/locales/vocabulary.js

```diff
@@ -99,47 +100,42 @@ export default {
   'editdialog.anonymousMessage2_login': 'log in to OpenStreetMap',
   'editdialog.anonymousMessage3': ', your changes will be immediate.',
   'editdialog.add_major_tag': 'Add',
-  'editdialog.location_checkbox': 'Change location',
+  'editdialog.location_checkbox': 'New location',
   'editdialog.location_placeholder': 'eg. across the street',
   'editdialog.location_editor_to_be_added': 'The position cannot be edited here yet, you can do so in the <a href="__link__">iD editor</a>.',
-  'editdialog.place_cancelled': 'Place permanently closed (delete)',
+  'editdialog.place_cancelled': 'Permanently closed (delete)',
   'editdialog.comment': 'Comment (optional)',
-  'editdialog.comment_placeholder': 'link to the source of information etc.',
+  'editdialog.comment_placeholder': 'Extra details, link to source, etc.',
   'editdialog.info_edit': `Your edit will be immediately saved to the OpenStreetMap. Please,
          enter only information from your own or verified sources. It is prohibited
          to copy copyrighted data (e.g. Google Maps). <a href="https://wiki.openstreetmap.org/wiki/How_We_Map">More info</a>`,
-  'editdialog.info_note': `Your suggestion will be processed by OpenStreetMap volunteers. Here
-         you can add an additional note or describe position adjustment etc.
-         It is also appropriate to support your contribution with a link to a source of information (web,
-         photo, etc.).`,
-  'editdialog.other_tags': 'Other properties – tags',
+  'editdialog.info_note': `Your suggestion will be reviewed by editors. You can add additional information for them below! If possible, please provide a link to the source that your suggestion is based on.`,
+  'editdialog.other_tags': 'Edit Tags',
   'editdialog.other_tags.new_key': 'new key',
-  'editdialog.other_tags.add': 'Add another tag',
+  'editdialog.other_tags.add': 'Add tag',
```

Comment:
This had a special meaning again. Its rather difficult to understand. If you fill in the key+value, you dont have to submit it. The button "Add another tag" is used for adding another row for another tag. 

I think by calling it "Add tag" it implies that is somehow adds the two fields.

Reply:
Maybe 
> New tag

I think the bigger issue here is the UX of adding tags itself.
Might be better if there was just a + icon honestly.

Reply:
Very good point. Lets go with the plus icon! 👍  I will resolve this comment and add the plus icon directly in code.